invenio_app_ils/patrons/api.py

```
@@ -166,6 +166,6 @@ def get_patron_or_unknown_dump(patron_pid):
     try:
         Patron = current_app_ils.patron_cls
         return Patron.get_patron(patron_pid).dumps_loader()
-    except:
+    except Exception as e:
```

bare except is a bad practice